src/iso15118/d20/state/dc_charge_loop.cpp

```diff
@@ -131,6 +149,13 @@ FsmSimpleState::HandleEventReturnType DC_ChargeLoop::handle_event(AllocatorType&
 
         ctx.respond(res);
 
+        if (req->display_parameters.has_value()) {
+            const auto& req_display_parameters = req->display_parameters.value();
+            const auto feedback_parameters = fill_parameters(req_display_parameters);
```

Comment:
would have used `const auto feedback_parameters = fill_parameters(*req->display_parameters)` if that works

Reply:
I'll try that out :+1: